client/main.lua

```diff
@@ -226,14 +226,24 @@ end
 ---@param args table<string, any>
 local function openVehicleCategoryMenu(args)
     local categoryMenu = {}
+    local sortedCategories = {}
     for k, v in pairs(config.shops[insideShop].categories) do
-        categoryMenu[#categoryMenu + 1] = {
-            title = v,
-            arrow = true,
-            onSelect = function()
-                openVehCatsMenu(k, args.targetVehicle)
+        table.insert(sortedCategories, v)
+    end
+    table.sort(sortedCategories)
+    for _, v in ipairs(sortedCategories) do
```

Comment:
Use a numeric for loop https://manason.github.io/effective-fivem-lua/tables/#use-numeric-for-loops-when-iterating-over-an-array